Resources/public/js/views/ez-contenteditview.js

```diff
@@ -210,6 +214,37 @@ YUI.add('ez-contenteditview', function (Y) {
          */
         _isTouch: function () {
             return Y.UA.touchEnabled;
+        },
+
+        /**
+         * Tap event handler on change language button. It fires `changeLanguage` event.
+         *
+         * @method _changeLanguage
+         * @private
+         * @param {EventFacade} e
+         */
+        _changeLanguage: function (e) {
+            e.preventDefault();
+
+            /**
+             * Fired when the change language link was tapped
+             *
+             * @event changeLanguage
+             */
+            this.fire('changeLanguage');
+        },
+
+        /**
+         * Sets language indicator
+         *
+         * @method setLanguageIndicator
+         * @param {String} languageCode
+         */
+        setLanguageIndicator: function (languageCode) {
```

Comment:
instead of this method, I would rather add a `languageCode` attribute to the content edit view and then update the indicator with the `languageCodeChange` event. That way, we clearly separate the data handling from the UI behaviour.